packages/thumbprint-react/rollup.config.js

```diff
@@ -2,6 +2,7 @@ import babel from 'rollup-plugin-babel';
 import path from 'path';
 import copy from 'rollup-plugin-cpy';
 import resolve from 'rollup-plugin-node-resolve';
+import typescript from 'rollup-plugin-typescript2';
```

Comment:
lol @ the package name

Reply:
Frontend infra is a respectable discipline and definitely not built on a load of paper clips duct-taped together.